src/main/webapp/static/js/addToCart.js

```diff
@@ -0,0 +1,32 @@
+function addToCart(buttons, cartListPlaces) {
+
+    for (let button of buttons) {
+        button.addEventListener('click', function (event) {
```

Comment:
Use named function for event listeners instead. Especially if it has multiple lines of code.